pkg/controller/nodemaintenance/taint.go

```diff
@@ -1,102 +0,0 @@
-package nodemaintenance
```

Comment:
As per @yanirq, we might want to hold off deleting this file.
Especially if kubevirt is relying on the existence of `kubevirt.io/drain`

Reply:
brought it back.